app/models/user.rb

```diff
@@ -28,6 +28,12 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
 
-  has_many :cohorts, class_name: 'Cohort', foreign_key: 'creator_id', dependent: :destroy
-  has_many :programs, class_name: 'Program', foreign_key: 'creator_id', dependent: :destroy
+  has_many :cohort_members, class_name: "CohortMember", foreign_key: "user_id", dependent: :destroy
```

Comment:
How would this one be used on user? There is another cohort_members in cohort that is used differently. I guess the name is confusing me or I just don't understand it 😂.

I guess my question is what will it return?